src/options.ts

```diff
@@ -1,9 +1,15 @@
+/** The possible image format */
+export type ImageFormat = 'image/png' | 'image/jpeg' | 'image/webp';
```

Comment:
Why is only let the type out?? I think easy to access like `TOptions['type']`

Reply:
Sorry, I don't quite understand what you mean.

Reply:
I think `ImageFormat` is not necessary. Because `ImageFormat` can be written like `TOptions['type']`.
Why did you think you needed `ImageFormat`?

Reply:
Ah, I think I see what you mean. The reason I exported the `ImageFormat` type alias was because I had written a wrapper around `getBase64Strings` to add some of my own custom defaults. Without the `ImageFormat` type, I wouldn't be able to strongly type the wrapper:

```ts
import { getBase64Strings, ImageFormat } from "exif-rotate-js";

async function resizeImage(
  file: File,
  maxSize: number,
  type: ImageFormat = "image/png",
  quality = 0.92,
): Promise<File> {
  const [resized] = await getBase64Strings([file], { maxSize, quality, type });

  // Simple way to read an image URL into a buffer
  const response = await fetch(resized);
  const data = [await response.blob()];

  return new File(data, file.name, { type });
}
```

I didn't see any reason why there would be a disadvantage in providing an explicit type for people to use.